resources/ts/Components/Integrations/Detail/CredentialsAuthClients.tsx

```diff
@@ -63,7 +64,19 @@ export const CredentialsAuthClients = ({
       </Heading>
       {keyVisibility === KeyVisibility.v1 ? (
         <div className="flex flex-col flex-1 gap-4">
-          <p>{t("details.credentials.uitid_alert")}</p>
+          <div>
+            <Trans
+              i18nKey="details.credentials.uitid_alert"
+              t={t}
+              components={[
+                <Link
+                  key={t("details.credentials.uitid_alert")}
```

Comment:
Is this key needed?

Reply:
Yes, otherwise you receive the error 'Missing "key" prop for element in array'.